### PR TITLE
Database connectivity

### PR DESCRIPTION
**Pull request by boostwilliam**

### CODE REVIEW

**Comment by boostwilliam** [comment]
Adds knex db client to payload of every resolver. I set up Knex to use environment variables specified in a .env file. I've included a .env.example file in source control so that developers may build their own.

File: server/handler.js [hunk 1]
```diff
@@ -3,6 +3,25 @@ const { typeDefs } = require('./graphql/typeDefs');
...
 const requireAuthDirective = require('./graphql/directives');
 const { playground } = require('./graphql/playground');
+const knexApi = require('knex');
+const dotenv = require('dotenv');
+
+// Reads in environment variables
+dotenv.config();
+
+// Acquires secret jwt signing
+const jwtSecret = process.env.JWT_SECRET;
```
Comment (hanseulshim) [inline server/handler.js]: Don't think we need JWT since we're pulling in login functionality from Blops.
Let's remove it for now.

Comment (boostwilliam) [reply server/handler.js]: Ah, a leftover. I'll remove it.

File: server/package.json [hunk 2]
```diff
@@ -9,8 +9,12 @@
   "dependencies": {
     "apollo-server-lambda": "^2.4.0",
     "aws-sdk": "^2.413.0",
+    "dotenv": "^7.0.0",
     "graphql": "^14.1.1",
-    "lodash.merge": "^4.6.1"
+    "jsonwebtoken": "^8.5.1",
```
Comment (hanseulshim) [inline server/package.json]: Uninstall JWT token as well

Comment (boostwilliam) [reply server/package.json]: Done

**Pull request closed without merging**